### PR TITLE
[WFCORE-5480] Bouncyclastle provider module requires a dependency on …

### PR DESCRIPTION
**Pull request by moulalis**
Issue: https://issues.redhat.com/browse/WFCORE-5480

### CODE REVIEW

**Comment by yersan** [comment]
Just adding hold label since I would like to avoid this PR create a conflict with the BC upgrade

File: core-feature-pack/common/src/main/resources/modules/system/layers/base/org/bouncycastle/bcprov/main/module.xml [hunk 1]
```diff
@@ -34,5 +34,9 @@
             <with-class name="org.bouncycastle.pqc.jcajce.provider.BouncyCastlePQCProvider"/>
         </service>
     </provides>
+    <dependencies>
+        <module name="java.sql"/>
+        <module name="java.naming"/>
+    </dependencies>
```
Comment (yersan) [inline core-feature-pack/common/src/main/resources/modules/system/layers/base/org/bouncycastle/bcprov/main/module.xml:40]: This is completely a nitpick and should not prevent this to be merged, but better if we keep the dependencies alphabetically when possible

**Review by yersan — APPROVED** [review]

**Comment by jmesnil** [comment]
@moulalis could you please rebase your PR, a conflict was introduced after we merged #4628

**Review by moulalis — COMMENTED** [review]
> @moulalis could you please rebase your PR, a conflict was introduced after we merged #4628

@jmesnil It's done. Thanks

**Comment by jmesnil** [comment]
@moulalis thanks

**Pull request merged**